app/main/views/briefs.py

```diff
@@ -151,6 +151,43 @@ def update_brief_add_user(brief_id, user):
     return jsonify(briefs=brief.serialize(with_users=True)), 200
 
 
+@main.route('/briefs/<int:brief_id>/suppliers/<int:supplier_code>', methods=['PUT'])
+def update_brief_add_rfx_seller(brief_id, supplier_code):
+    updater_json = validate_and_return_updater_request()
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+
+    # Find the supplier that matches the inputted code, otherwise 404.
+    supplier = Supplier.query.filter(
+        Supplier.code == supplier_code
+    ).first_or_404()
+
+    if not supplier:
+        raise ValidationError("No supplier found: " + supplier_code)
+    # Assign the current seller data to separate variable.
+    # Add the found supplier to said variable.
+    # Amend the brief's data to reflect the added supplier.
+    sellers = brief.data['sellers']
+    sellers[str(supplier_code)] = {'name': supplier.name}
+    brief.data['sellers'] = sellers
+    audit = AuditEvent(
+        audit_type=AuditTypes.update_brief,
```

Comment:
it might be better to add a new type of audit for this.

Reply:
This has been updated in my latest commit.